### ISSUE TITLE
Is it possible to provide train/val loss vs epochs graph?

### ISSUE THREAD

**Issue by HarshavardhanaTG** [facebookresearch/emg2qwerty#67]
Is it possible to provide a train/val loss vs. epoch graph for a couple of users (say, user 0, 6, or 7 in the test set) when starting with random initialization? I think I have set up most of the code correctly, but many times, the model does not converge even after adding weight_decay with random initialization. That would help me debug.

Thanks.

**Comment by HarshavardhanaTG** [comment]
Never mind. Got it.
